plugins/tech-insights-backend-module-jsonfc/config.json

```diff
@@ -0,0 +1,183 @@
+{
+  "$schema": "https://backstage.io/schema/config-v1",
+  "type": "object",
+  "properties": {
+    "techInsights": {
+      "type": "object",
+      "properties": {
+        "factChecker": {
+          "type": "object",
+          "properties": {
+            "checks": {
+              "type": "object",
+              "additionalProperties": {
+                "$ref": "#/$defs/check"
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "$defs": {
+    "check": {
+      "type": "object",
+      "required": ["type", "name", "description", "factIds", "rule"],
```

Comment:
I think the description probably shouldn't be a required field. I don't know why I made it so originally but maybe we can backtrack on that decision.

Reply:
Hard to judge for me.

We could make `TechInsightCheck.description` optional. This would be a compatible change from the producer perspective. However, there might be places that assume it is required.

E.g., at `plugin-tech-insights` `ScorecardsList`

```ts
<MarkdownContent content={result.check.description} />
```
used as a fallback and expecting `string` and not `undefined`.

In general, I would rather keep this as a separate change before or after this PR.

Reply:
Makes sense as a separate item